ext/intl/grapheme/grapheme_util.c

```diff
@@ -387,8 +385,12 @@ UBreakIterator* grapheme_get_break_iterator(void *stack_buffer, UErrorCode *stat
 		INTL_G(grapheme_iterator) = global_break_iterator;
 	}
 
-	buffer_size = U_BRK_SAFECLONE_BUFFERSIZE;
+#if U_ICU_VERSION_MAJOR_NUM > 70
```

Comment:
Apparently, [`ubrk_safeClone()` is already deprecated as of ICU 69](https://unicode-org.github.io/icu-docs/apidoc/dev/icu4c/ubrk_8h.html#a6901f59d9c74bc8e16b9994cb1be7679), so:
```suggestion
#if U_ICU_VERSION_MAJOR_NUM >= 69
```

Reply:
Problem is however the replacement calls do not exist yet (I have locally the 70.x release).

Reply:
ah I see in fact they re "conditionals"